frontend/src/features/permits/components/form/ContactDetails.tsx

```diff
@@ -110,7 +152,13 @@ export const ContactDetails = ({ feature }: { feature: string }) => {
           feature={feature}
           options={{
             name: "permitData.contactDetails.additionalEmail",
-            rules: { required: false },
+            rules: {
+              required: false,
+              validate: {
+                validateEmail: (email: string) =>
```

Comment:
Since the additional email is optional, this can be:
```
validateEmail: (email?: string) =>
    !email || (email.length === 0) || isEmail(email) || invalidEmail(),
```